.gitignore

```diff
@@ -24,7 +24,11 @@
 # Ignore the following directories and file extensions
 target
 generated-docs
+
+# todo remove after upgrade to zig 0.13.0
```

Comment:
In the transition it is helpful to keep both the old and new cache locations ignored so users switching between these branches are less affected.